books.html

```diff
@@ -0,0 +1,2250 @@
+
+
+
+
+
+
+    
+        <p class="price_color">£35.02</p>
+    
+
+<p class="instock availability">
+    <i class="icon-ok"></i>
+    
+        In stock
+    
+</p>
+
+                
+                    
+
+
+
+
+
+
+    
+    <form>
+        <button type="submit" class="btn btn-primary btn-block" data-loading-text="Adding...">Add to basket</button>
+    </form>
+
+
+                
+            </div>
+        
+    </article>
+
+</li>
+                    
+                        <li class="col-xs-6 col-sm-4 col-md-3 col-lg-3">
+
+
+
+
+
+
+    <article class="product_pod">
+        
+            <div class="image_container">
+                
+                    
+                    <a href="../../our-band-could-be-your-life-scenes-from-the-american-indie-underground-1981-1991_985/index.html"><img src="../../../media/cache/54/60/54607fe8945897cdcced0044103b10b6.jpg" alt="Our Band Could Be Your Life: Scenes from the American Indie Underground, 1981-1991" class="thumbnail"></a>
+                    
+                
+            </div>
+        
+
+        
+            
+                <p class="star-rating Three">
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                </p>
+            
+        
+
+        
+            <h3><a href="../../our-band-could-be-your-life-scenes-from-the-american-indie-underground-1981-1991_985/index.html" title="Our Band Could Be Your Life: Scenes from the American Indie Underground, 1981-1991">Our Band Could Be ...</a></h3>
+        
+
+        
+            <div class="product_price">
+                
+
+
+
+
+
+
+    
+        <p class="price_color">£57.25</p>
+    
+
+<p class="instock availability">
+    <i class="icon-ok"></i>
+    
+        In stock
+    
+</p>
+
+                
+                    
+
+
+
+
+
+
+    
+    <form>
+        <button type="submit" class="btn btn-primary btn-block" data-loading-text="Adding...">Add to basket</button>
+    </form>
+
+
+                
+            </div>
+        
+    </article>
+
+</li>
+                    
+                        <li class="col-xs-6 col-sm-4 col-md-3 col-lg-3">
+
+
+
+
+
+
+    <article class="product_pod">
+        
+            <div class="image_container">
+                
+                    
+                    <a href="../../olio_984/index.html"><img src="../../../media/cache/55/33/553310a7162dfbc2c6d19a84da0df9e1.jpg" alt="Olio" class="thumbnail"></a>
+                    
+                
+            </div>
+        
+
+        
+            
+                <p class="star-rating One">
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
```

Comment:
The star-rating class 'One' does not match the number of filled stars (five). This inconsistency might confuse users or affect the visual representation of ratings.